build/webpack/configs/base.config.ts

```diff
@@ -19,6 +19,7 @@ export const baseConfig = {
       '~modules': `${BASE_DIR}/modules`,
       '~utils': `${BASE_DIR}/src/components/utils`,
       '~supported-documents': `${BASE_DIR}/src/supported-documents`,
+      '~WorkflowEngine': `${BASE_DIR}/src/WorkflowEngine`,
```

Comment:
Everything seems to be is kebab case. Maybe you should try to rename to 'workflow-engine'

Reply:
done